### PR TITLE
Immutable admin [BE]

### LINKED ISSUE #425: Set default admin
There needs to be a default admin user. This user will always remain the 'super' admin and cannot be removed.

### PR DESCRIPTION
**Pull request by jay-tux**
Fixes #425

### CODE REVIEW

File: backend/utility.ts [hunk 1]
```diff
@@ -511,3 +514,9 @@ export function queryToBody(req: express.Request) {
     }
     return req;
 }
+
+export function mutable<T>(obj: T, targetid: number): Promise<T> {
+    if (targetid == config.global.defaultUserId)
```
Comment (BramDevlaminck) [inline backend/utility.ts:519]: We should probably use `===` instead of `==`

**Review by BramDevlaminck — APPROVED** [review]
seems to work for me! awesome!

File: backend/utility.ts (hunk 1 above)
Comment (jay-tux) [reply backend/utility.ts:519]: Will do

**Review by KoenDesplenter — APPROVED** [review]
Looks great

**Pull request merged**